theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -314,11 +317,12 @@ def fn(x):
             # Test the GPU grad + GPU implementation
             def fn(x):
                 dnn_op = cuda.dnn.dnn_pool(
-                    x, ws=(ws, ws),
-                    stride=(stride, stride),
-                    pad=pad,
+                    x, ws=ws,
+                    stride=stride,
+                    pad=pad_,
```

Comment:
We need both to be tested. But I would go to make this test that is more complete test the new code and make a new test, that test only 1 case with the old interface. We just need to make sure the old interface work.